client/src/containers/transactions/components/form/index.tsx

```diff
@@ -0,0 +1,364 @@
+/* eslint @typescript-eslint/indent: 0,
+   react/jsx-curly-newline: 0 */ // Linter and prettier conflicts, should be fixed.
```

Comment:
It will never be fixed XD

Reply:
tikriausiai tiesa, bet čia reik laiko, kurio nelabai turime ir nėra pirmo būtinumo, nes arba vienas arba kitas pasirūpina vis tiek.